tutorials/notebooks/keras/ptq/example_keras_activation_z_score_threshold.ipynb

```diff
@@ -0,0 +1,847 @@
+      },
+      "source": [
+        "Finally we quantize the model, this can take some time. Grab a coffee!"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "ba0c6e55-d474-4dc3-9a43-44b736635998",
+      "metadata": {
+        "id": "ba0c6e55-d474-4dc3-9a43-44b736635998"
+      },
+      "outputs": [],
+      "source": [
+        "quantized_models_dict = {}\n",
+        "\n",
+        "for z_score, q_config in q_configs_dict.items():\n",
+        "    # Create a CoreConfig object with the current quantization configuration\n",
+        "    ptq_config = mct.core.CoreConfig(quantization_config=q_config)\n",
+        "\n",
+        "    # Perform MCT post-training quantization\n",
+        "    quantized_model, quantization_info = mct.ptq.keras_post_training_quantization(\n",
+        "        in_model=float_model,\n",
+        "        representative_data_gen=representative_dataset_gen,\n",
+        "        core_config=ptq_config,\n",
+        "        target_platform_capabilities=tpc\n",
+        "    )\n",
+        "\n",
+        "    # Update the dictionary to include the quantized model\n",
+        "    quantized_models_dict[z_score] = {\n",
+        "        \"quantization_config\": q_config,\n",
+        "        \"quantized_model\": quantized_model,\n",
+        "        \"quantization_info\": quantization_info\n",
+        "    }\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "A8UHRsh2khM4",
+      "metadata": {
+        "id": "A8UHRsh2khM4"
+      },
+      "source": [
+        "### Z-Score Threshold and Distribution Visulisation"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "Y-0QLWFJkpFV",
+      "metadata": {
+        "id": "Y-0QLWFJkpFV"
+      },
+      "source": [
+        "To assist with understanding we will now plot the activation distribution of Mobilenet's first activation layer.\n",
+        "\n",
+        "This will be obtained by feeding the representative dataset through the model.\n",
+        "To see the distribution of the activations the model needs to be rebuilt upto and including the layer chosen for distribution visulisation.\n",
+        "\n",
+        "To see said layers z-score threshold values. we will need to calculate these manually using the equestion stated in the introduction.\n",
+        "\n",
+        "To plot the distribution we first need to list the layer names. With keras this can be done easily using the following. We established the index of the layer of interest using various checks that can be seen in the appendix section."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac",
+      "metadata": {
+        "id": "a22e6d68-c40f-40bf-ab74-ff453011aeac"
+      },
+      "outputs": [],
+      "source": [
+        "#print layer name\n",
+        "for index, layer in enumerate(float_model.layers):\n",
```

Comment:
wouldn't `print(float_model.layers[51].name)` do the trick?

Reply:
?

Reply:
sorry, miss read your original comment. fixed